data/deep missions.txt

```diff
@@ -449,16 +446,24 @@ mission "Deep: Mystery Cubes 3: Escorts"
 				"Raven" 2
 				"Dagger"
 	npc
-		personality heroic opportunistic escort vindictive
 		government Republic
+		personality heroic opportunistic escort vindictive
 		fleet
 			names "republic capital"
-			fighters "republic fighter"
 			variant
 				"Cruiser"
 				"Gunboat" 2
+	npc
+		government Republic
+		personality timid
+		fleet
+			fighters "republic fighter"
+			variant
 				"Combat Drone" 4
-	
+
+# Should there be a mission with a specific DSS belonging to Paris that is `accompany save` and
+# `personality timid escort`? It's possible (though very unlikely) that the escorts above get
+# killed by the DMC3 opponents and naturally spawning pirates.
```

Comment:
Unsure if this should be the case. I'll look into it during play testing.

Reply:
I really don't think it is likely that the player and the mission npcs will be defeated by 3 furys and 2 cruisers, but given that we get the DSS Faraday later, I thought it might be sensible to have it show up here too